### PR TITLE
docs(config): add environment variables for Cube Store minIO support

### PR DESCRIPTION
**Pull request by PieterVanZyl-Dev**
**Check List**
- [X] Tests has been run in packages where changes made if available (Not Applicable)
- [X] Linter has been run for changed code (Not Applicable)
- [X] Tests for the changes have been added if not covered yet (Not Applicable)
- [X] Docs have been added / updated if required

### CODE REVIEW

File: docs/content/Configuration/Environment-Variables-Reference.md [hunk 1]
```diff
@@ -176,6 +176,11 @@ databases [in this guide][link-connecting-to-db].
...
+| `CUBESTORE_MINIO_ACCESS_KEY_ID`            | The Access Key ID for minIO. Required when using minIO                                                                                  | A valid minIO Access Key ID                                        |
+| `CUBESTORE_MINIO_SECRET_ACCESS_KEY`        | The Secret Access Key for minIO. Required when using minIO                                                                              | A valid minIO Secret Access Key                                     |                                                           |
+| `CUBESTORE_MINIO_BUCKET`                    | The name of the bucket that you want to use minIO. Required when using minIO                                                                             | A valid bucket name in the AWS account                                                  |
+| `CUBESTORE_MINIO_SERVER_ENDPOINT`                    | The minio server endpoint. Required when using minIO                                                                            | A valid minIO endpoint eg: `http://localhost:9000`                                            |
```
Comment (hassankhan) [inline docs/content/Configuration/Environment-Variables-Reference.md:182]:
```suggestion
| `CUBESTORE_MINIO_SERVER_ENDPOINT`                    | The minIO server endpoint. Required when using minIO                                                                            | A valid minIO endpoint eg: `http://localhost:9000`                                            |
```

**Review by hassankhan — CHANGES_REQUESTED** [review]
Hi @PieterVanZyl-Dev :wave:

Thanks so much for the documentation PR, just had one minor nitpick

File: docs/content/Configuration/Environment-Variables-Reference.md (hunk 1 above)
Comment (PieterVanZyl-Dev) [reply docs/content/Configuration/Environment-Variables-Reference.md:182]: @hassankhan Happy with those changes you can commit and complete if there isn't anything else

Thanks for the help, review 🎉

**Review by hassankhan — APPROVED** [review]

**Comment by hassankhan** [comment]
Thanks for your contribution @PieterVanZyl-Dev, much appreciated 💯

**Pull request merged**